pages/k8s/upgrade-notes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Upgrade notes | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, upgrading, track, version
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: upgrade-notes.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+# Upgrade notes
+
+This page is intended to deal with specific, special circumstances you may
+encounter when upgrading between versions of  the **Canonical Distribution of
+Kubernetes<sup>&reg;</sup>.** The notes are organised according to the upgrade
+path below, but also be aware that any upgrade that spans more than one minor
+version may need to beware of notes in any of the intervening steps.
+
+## Upgrading from 1.9.x to 1.10.x
```

Comment:
This is the only issue I am aware of currently. Still, it seemed to be better to split this off on a separate page rather than stick it in the middle of the main upgrade page.